components/system-api/src/private.rs

```diff
@@ -16,6 +16,188 @@
+//!
+//! # List Peers
+//!
+//! | Property    | Value |
+//! |-------------|-------|
+//! | Path        | `/api/system/v1/peers` |
+//! | Method      | GET   |
+//! | Query type  | - |
+//! | Return type | [`PeersInfo`] |
+//!
+//! Returns the list of incoming and outgoing connections for current node.
+//!
+//! [`PeersInfo`]: struct.PeersInfo.html
+//!
+//! ```
+//! use exonum_system_api::{private::PeersInfo, SystemApiPlugin};
+//! use exonum_testkit::{ApiKind, TestKitBuilder};
+//!
+//! # fn main() -> Result<(), failure::Error> {
+//! let mut testkit = TestKitBuilder::validator()
+//!     .with_plugin(SystemApiPlugin)
+//!     .create();
+//! let api = testkit.api();
+//! let info: PeersInfo = api.private(ApiKind::System).get("v1/peers")?;
+//! # Ok(())
+//! # }
+//! ```
+//!
+//! # Add Peer
+//!
+//! | Property    | Value |
+//! |-------------|-------|
+//! | Path        | `/api/system/v1/peers` |
+//! | Method      | POST   |
+//! | Query type  | [`ConnectInfo`] |
+//! | Return type | - |
+//!
+//! Adds a peer to the Exonum node. Node will attempt to connect to this peer.
+//!
+//! [`ConnectInfo`]: https://docs.rs/exonum-node/latest/exonum_node/struct.ConnectInfo.html
+//!
+//! ```
+//! use exonum_node::ConnectInfo;
+//! use exonum_system_api::SystemApiPlugin;
+//! use exonum_testkit::{ApiKind, TestKitBuilder};
+//!
+//! # fn main() -> Result<(), failure::Error> {
+//! # let address = "127.0.0.1:8080".to_owned();
+//! # let public_key = Default::default();
+//! // Obtaining address and public key of target node skipped...
+//! let connect_info = ConnectInfo {
+//!     address,
+//!     public_key,
+//! };
+//!
+//! let mut testkit = TestKitBuilder::validator()
+//!     .with_plugin(SystemApiPlugin)
+//!     .create();
+//! let api = testkit.api();
+//! api.private(ApiKind::System)
+//!     .query(&connect_info)
+//!     .post("v1/peers")?;
+//! # Ok(())
+//! # }
+//! ```
+//!
+//! # Get Node Info
+//!
+//! | Property    | Value |
+//! |-------------|-------|
+//! | Path        | `/api/system/v1/network` |
+//! | Method      | GET   |
+//! | Query type  | - |
+//! | Return type | [`NodeInfo`] |
+//!
+//! Obtains information about node.
+//!
+//! [`NodeInfo`]: struct.NodeInfo.html
+//!
+//! ```
+//! use exonum_system_api::{private::NodeInfo, SystemApiPlugin};
+//! use exonum_testkit::{ApiKind, TestKitBuilder};
+//!
+//! # fn main() -> Result<(), failure::Error> {
+//! let mut testkit = TestKitBuilder::validator()
+//!     .with_plugin(SystemApiPlugin)
+//!     .create();
+//! let api = testkit.api();
+//! let info: NodeInfo = api.private(ApiKind::System).get("v1/network")?;
+//! # Ok(())
+//! # }
+//! ```
+//!
+//! # Check if Consensus is Enabled
+//!
+//! | Property    | Value |
+//! |-------------|-------|
+//! | Path        | `/api/system/v1/consensus_enabled` |
+//! | Method      | GET   |
+//! | Query type  | - |
+//! | Return type | `bool` |
+//!
+//! Returns `true` if consensus is enabled on the node, and `false` otherwise.
```

Comment:
Nit / future work: It could be worth describing the difference with the consensus info returned by the `healthcheck` endpoint.